modules/apps/admin-apps-page.php

```diff
@@ -93,18 +93,18 @@ private static function filter_wporg_app( $app ) {
 
 		if ( static::is_plugin_installed( $app['file_path'] ) ) {
 			if ( current_user_can( 'activate_plugins' ) ) {
-				$app['action_label'] = 'Activate';
+				$app['action_label'] = __( 'Activate', 'elementor' );
```

Comment:
This needs to be esc_html__ and not just __
To avoid HTML injections in translations.

Reply:
```suggestion
				$app['action_label'] = esc_html__( 'Activate', 'elementor' );
```